rust/main/agents/scraper/src/db/generated/prelude.rs

```diff
@@ -1,4 +1,4 @@
-//! `SeaORM` Entity. Generated by sea-orm-codegen 0.12.3
+//! `SeaORM` Entity, @generated by sea-orm-codegen 1.1.1
 #[allow(unused_imports)]
 pub use super::{
     block::Entity as Block, cursor::Entity as Cursor,
```

Comment:
I suppose we can commit prelude which was generated by `generate_entities`.

Reply:
I am not sure what you mean by `commit` could you please specify?

Reply:
If you run `generate_entities`, it will generate entities files. They will be the same as currently committed in this PR with the exception of `prelude.rs` file. I think that `prelude.rs` can be also added to this PR. Current one and the generate one are equivalent.

Reply:
Ahhh, I get it now, the new prelude is added in this PR. It is just that the generated one has a line by line format, not a block format, I changed it back to the block format so we can use the `allow_unused_imports` on the whole block as before. You can see the update I did here https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4925/commits/ed6d556bf56f700fb536bcc4045f7cc1fd168176

But you are right, the generated one is the same as the old one so everything is fine in that regard

Reply:
I see it now. We need to add `allow_unused_imports` anyway and it gets removed by generation script. So, this file will change no matter what. Block imports are fine then.